_scripts/dev-runner.js

```diff
@@ -51,10 +51,12 @@ async function restartElectron() {
 
   electronProcess = spawn(electron, [
     path.join(__dirname, '../dist/main.js'),
-    // '--enable-logging', Enable to show logs from all electron processes
+    // '--enable-logging', // Enable to show logs from all electron processes
     remoteDebugging ? '--inspect=9222' : '',
     remoteDebugging ? '--remote-debugging-port=9223' : '',
-  ])
+  ],
+    // { stdio: 'inherit' } // required for logs to actually appear in the stdout
+  )
```

Comment:
What are these changes?

Reply:
`{ stdio: 'inherit' }` makes `spawn()` pipe the stdout and stderr of the spawned command to the stdout and stderr of the parent process, in this case `_scripts/dev-runner.js`. Without that change you won't see any logs from the `main` process, I've left it in there commented out so that if someone needs to log something from the `main` process in the future they don't need to spend hours trying to work out why they can't see the logs, like I did.

Adding the extra `//` to the enable logging line, makes it easier to use your IDE's keyboard shortcuts to uncomment the line without needing to go and add the slashes in manually after you have uncommented it. As `Enable to show logs from all electron processes` is not valid javascript code and will need to stay commented even when you uncomment that line.